src/service_worker.cpp

```diff
@@ -146,18 +168,18 @@ bool ServiceWorker::checkKinematicsQuery(tub_feasibility_check::CheckKinematics:
   WorldCollisionTypes world_collision_types(part_to_type);
 
   ROS_INFO("Setting ifco pose and creating bounding boxes");
-  ifco_scene->moveIfco(ifco_transform);
-  ifco_scene->removeBoxes();
+  ifco_scene_->moveIfco(ifco_transform);
```

Comment:
Why do you need to change the name? This change is not required for functionality porous but increases the diff. Please revert the name.

Reply:
This was addressed in f1df164201f6898fdea2a450a777df03796e125b.